docs/writing-rules/testing-rules.md

```diff
@@ -60,13 +63,13 @@ Run the tests with the following:
 $ python -m semgrep --quiet --test rules/
 1 yaml files tested
 check id scoring:
-================================================================================
+--------------------------------------------------------------------------------
```

Comment:
Nit: I changed the output here from `=` -> `-` for consistency with other outputs in the CLI :man_shrugging: